src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -690,6 +690,7 @@ def launch_fluent(
                     cleanup_on_exit=cleanup_on_exit,
                     start_transcript=start_transcript,
                     launcher_args=argvals,
+                    inside_container=True,
```

Comment:
In all other scenarios FluentConnection will have it that inside_container=False, which might not always be correct.

Reply:
That is true. If the user for example manually starts up the Fluent docker container without using PyFluent, gets the proper connection information, and sets up the connection to PyFluent using `FluentConnection(..., inside_container=False)`, which is the default, the `FluentConnection` will work, but `force_exit` won't work. Ideally the user would correctly specify that `inside_container=True`. 

I am not aware of other scenarios where the current set up wouldn't work.

I don't currently know of a better way to tell whether Fluent is running inside a container or not. Going to investigate this. @mkundu1 perhaps any ideas that come to mind?

Regardless of whether we find a better way to check if inside container or not, for now I am going to change `inside_container` default from `False` to `None`, which I believe better passes the information that it hasn't been specified whether it is inside a container or not.